demosplan/DemosPlanCoreBundle/Logic/ProcedureStatisticsService.php

```diff
@@ -76,27 +81,29 @@ private function getSegmentStatistic(string $statistic, string $procedureId): in
             ->andWhere('s NOT INSTANCE OF '.Segment::class)
             ->setParameter('procedureId', $procedureId);
 
-        if ('statementsWithSegments' === $statistic) {
+        if ('newStatements' === $statistic) {
             $query->andWhere(
                 $qb->expr()
                     ->in('s.id', $segmentQuery->getDQL())
             );
         }
 
-        if ('statementsWithoutSegments' === $statistic) {
+        if ('inProgressSegments' === $statistic) {
             $query->andWhere(
                 $qb->expr()
-                    ->notIn('s.id', $segmentQuery->getDQL())
+                    ->notIn('s.id', $segmentQuery
+                        ->andWhere('place.solved = 0')
```

Comment:
is there  andWhereAny?
If so, shouldn't it be something like 
```suggestion
                    ->notIn('s.id', $segmentQuery
                        ->andWhereAny('place.solved = 1')
```